impl_core/src/test/java/io/opencensus/implcore/metrics/MetricRegistryImplTest.java

```diff
@@ -263,9 +263,9 @@ public long applyAsLong(Object value) {
                     Type.GAUGE_INT64,
                     Collections.unmodifiableList(Collections.singletonList(LABEL_KEY))),
                 Collections.singletonList(
```

Comment:
We may want to add the same trick in the Metric for TimeSeries. Not now but we can keep this in mind.